scripts/verify.sh

```diff
@@ -14,10 +14,12 @@ else
 fi
 GOVCMS_VERSION=$(cat ./satis-config/${config} | jq -r '.require."govcms/govcms"')
```

Comment:
Missing GOVCMS_VERSION_SCAFFOLD_TOOLING and GOVCMS_VERSION_REQUIRE_DEV in here

```suggestion
GOVCMS_VERSION=$(cat ./satis-config/${config} | jq -r '.require | .["govcms/govcms"]')
# Versions needed for stable testing.
GOVCMS_VERSION_SCAFFOLD_TOOLING=$(cat ./satis-config/"${config}" | jq -r '.require | .["govcms/scaffold-tooling"]')
GOVCMS_VERSION_REQUIRE_DEV=$(cat ./satis-config/"${config}" | jq -r '.require | .["govcms/require-dev"]')
```

Reply:
Ahh i thought my verify passed, it clearly didn't. Un momento.